packages/@aws-cdk/assertions/lib/template.ts

```diff
@@ -21,34 +21,39 @@ export class Template {
   /**
    * Base your assertions on the CloudFormation template synthesized by a CDK `Stack`.
    * @param stack the CDK Stack to run assertions on
+   * @param skipCyclicalDependenciesCheck Optional param to support unprocessed circular transform templates
    */
-  public static fromStack(stack: Stack): Template {
-    return new Template(toTemplate(stack));
+  public static fromStack(stack: Stack, skipCyclicalDependenciesCheck?: boolean): Template {
```

Comment:
Put this property in a `TemplateParsingOptions` struct or something.

Reply:
Sure :)